testdata/goldens/ruby/Google-Api-FieldBehavior.html

```diff
@@ -0,0 +1,25 @@
+﻿<!DOCTYPE html>

```

Comment:
Unrelated to adding goldens. This appears to be a generic file that isn't specific to Vision? If so, is there a way we can avoid this duplication for every package?

Reply:
possibly. They are currently generated for every package on googleapis.dev, and rubydoc.info, so I'd like to leave that as is, for now at least.